src/common/intl.cpp

```diff
@@ -223,264 +225,22 @@ const wxLanguageInfos& wxGetLanguageInfos()
     }
 }
 
-
-void wxLocale::DoCommonInit()
```

Comment:
I think this code was just moved, without changes, but GitHub UI doesn't have the equivalent of Git `--color-moved`, so I'm not sure -- please let me know if there were any changes worth being reviewed here.

Reply:
Yes, the code was simply _moved_ **without any changes**. 

Maybe a later PR will introduce changes, but no changes for now.